pkg/genevalogging/genevalogging.go

```diff
@@ -362,9 +372,15 @@ func (g *genevaLogging) CreateOrUpdate(ctx context.Context) error {
 			Selector: &metav1.LabelSelector{
 				MatchLabels: map[string]string{"app": "mdsd"},
 			},
+			UpdateStrategy: appsv1.DaemonSetUpdateStrategy{
+				Type: appsv1.RollingUpdateDaemonSetStrategyType,
+				RollingUpdate: &appsv1.RollingUpdateDaemonSet{
+					MaxUnavailable: &maxUnavailablePods,
+				},
+			},
 			Template: v1.PodTemplateSpec{
 				ObjectMeta: metav1.ObjectMeta{
-					Labels:      map[string]string{"app": "mdsd"},
+					Labels:      map[string]string{"app": "mdsd", "certHash": certHash},
```

Comment:
i do this in the operator, just differently: https://github.com/Azure/ARO-RP/blob/aro-operator/pkg/controllers/genevalogging_controller.go#L109
https://github.com/Azure/ARO-RP/blob/aro-operator/pkg/dynamichelper/workloadhelper.go